packages/react-ui/src/widgets/RangeWidgetUI.js

```diff
@@ -138,35 +149,50 @@ function RangeWidgetUI({ data, min, max, limits, onSelectedRangeChange }) {
     onSelectedRangeChange && onSelectedRangeChange(newValues);
   };
 
+  const resetSlider = () => {
```

Comment:
I think there is a missing call to `onSelectedRangeChange` here, otherwise it's just resetting the UI state but not the underlying filter. And I would expect `onSelectedRangeChange` to simply remove the filter in this case.